src/workflows/Opportunity.workflow

```diff
@@ -63,6 +63,10 @@
             <operation>notEqual</operation>
             <value>0</value>
         </criteriaItems>
+        <criteriaItems>
+            <field>Opportunity.Fair_Market_Value__c</field>
```

Comment:
Can Fair_Market_Value__c ever be negative? If so, do we still want it to copy over to Amount?

Reply:
I guess that would have happened before this update, tho Judi suggested using `(Opportunity: Fair Market Value GREATER THAN 0)` in the issue